packages/manager/src/features/ObjectStorage/BucketLanding/BucketRateLimitTable.tsx

```diff
@@ -83,7 +83,7 @@ export const BucketRateLimitTable = ({
           'This endpoint type supports up to 750 Requests Per Second (RPS). '
         )}
         Understand{' '}
-        <Link to="https://techdocs.akamai.com/cloud-computing/docs/create-and-manage-buckets">
+        <Link to="https://techdocs.akamai.com/cloud-computing/docs/object-storage#optimize-applications-to-avoid-rate-limiting">
```

Comment:
Seems like this link might be better?

```suggestion
        <Link to="https://techdocs.akamai.com/cloud-computing/docs/object-storage-product-limits">
```

Reply:
Hmm that's weird... they must have moved the docs recently